components/builder-web/app/actions/index.ts

```diff
@@ -193,13 +200,13 @@ export const signOut = usersActions.signOut;
 
 export const setLayout = uiActions.setLayout;
 
-export const getCookie    = cookieActions.getCookie;
-export const setCookie    = cookieActions.setCookie;
+export const getCookie = cookieActions.getCookie;
+export const setCookie = cookieActions.setCookie;
```

Comment:
![tenor-119640324](https://user-images.githubusercontent.com/223407/30627996-98e9dbb6-9e0e-11e7-957f-e9101c765b0e.gif)

Reply:
![tenor-112075757](https://user-images.githubusercontent.com/1130349/30628525-dcc1dc32-9e11-11e7-8598-6293d2ca555e.gif)

I blame the auto-save linter